apps/aehttp/test/aehttp_hyperchains_SUITE.erl

```diff
@@ -1095,12 +1116,12 @@ mine_key_blocks(ParentNodeName, NumParentBlocks) ->
     ct:log("Parent block mined ~p ~p number: ~p", [KBs, ParentNodeName, NumParentBlocks]),
     {ok, KBs}.
 
-get_block_producer_name(Parties, Node, Height) ->
-    Producer = get_block_producer(Node, Height),
-    case lists:keyfind(Producer, 1, Parties) of
-        false -> Producer;
-        {_, _, Name} -> Name
-    end.
+%get_block_producer_name(Parties, Node, Height) ->
```

Comment:
Why not deleted?

Reply:
I believe I'll need it if I restore the other tests